src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1663,9 +1663,21 @@ bool AdaptDispatch::SoftReset()
 // True if handled successfully. False otherwise.
 bool AdaptDispatch::HardReset()
```

Comment:
Should we add this to the doc comment for this function?

Reply:
Yeah, that seems like a good idea. I've added something in commit 8003b5265d99c89323d3713b82c3075fc43c37cc. Sorry for the delay, but it's been a long day at work.